api/prisma/seed.ts

```diff
@@ -101,49 +101,35 @@ export const initialDevSeed = async (): Promise<void> => {
     }
 
     if (process.env.STAGE === 'local') {
-        // create S3 bucket locally for image uploads
-
-        try {
-            await s3.client.send(
-                new GetBucketAclCommand({
-                    Bucket: `science-octopus-publishing-images-${process.env.STAGE}`
-                })
-            );
-            console.log('Bucket already exists');
-        } catch (err) {
-            // Bucket does not exist, therefor create
-            await s3.client.send(
-                new CreateBucketCommand({
-                    Bucket: `science-octopus-publishing-images-${process.env.STAGE}`
-                })
-            );
-            console.log('Bucket created');
-        }
-
-        // create S3 bucket locally for PDF uploads
-        try {
-            await s3.client.send(
-                new GetBucketAclCommand({
-                    Bucket: `science-octopus-publishing-pdfs-${process.env.STAGE}`
-                })
-            );
-            console.log('Bucket already exists');
-        } catch (err) {
-            // Bucket does not exist, therefor create
-            await s3.client.send(
-                new CreateBucketCommand({
-                    Bucket: `science-octopus-publishing-pdfs-${process.env.STAGE}`
-                })
-            );
-            console.log('Bucket created');
+        // Create local S3 buckets
+        for (const bucketNameSegment of ['images', 'pdfs', 'sitemaps']) {
```

Comment:
You can do `for (const bucketNameSegment in s3.buckets) { ...` instead of the hardcoded array

Reply:
That's better, thanks, done